src/arbitrary.rs

```diff
@@ -131,9 +131,7 @@ pub trait Arbitrary: Clone + 'static {
 }
 
 impl Arbitrary for () {
-    fn arbitrary(_: &mut Gen) -> () {
-        ()
-    }
+    fn arbitrary(_: &mut Gen) {}
```

Comment:
I think I prefer how it was written since it's an impl for `()` specifically.

Reply:
Oh, I see! I'll roll it back then.